using-mobile-extensions/adobe-campaign-standard/README.md

```diff
@@ -188,6 +188,8 @@ If you are developing an Android application, to correctly display fullscreen in
 ```markup
 <activity android:name="com.adobe.marketing.mobile.FullscreenMessageActivity" />
 ```
+In addition to this, a global lifecycle callback must be defined in your app's MainActivity to ensure the proper display of fullscreen in-app messages. The instructions to do this can be seen at: [Implementing Global Lifecycle Callbacks](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle/lifecycle-extension-in-android#implementing-global-lifecycle-callbacks) 
```

Comment:
- Not sure what "this" refers to in the first sentence. We should add a noun after "this".

- Please reword the 2nd sentence so that it reads as follows:

"To define the global lifecycle callback, see [Implementing..."

- Add a period at the end of the 2nd sentence.